packages/manager/modules/pci/src/components/project/regions-list/regions-list.controller.js

```diff
@@ -75,6 +72,16 @@ export default class RegionsListController {
     }
   }
 
+  static isRegionDisabled(region) {
```

Comment:
```suggestion
  static isRegionDisabled([region]) {
      return !get(region, 'hasEnoughQuota', false);
  }
```
but I don't think this is necessarily better